connector/avro/src/main/scala/org/apache/spark/sql/avro/SchemaConverters.scala

```diff
@@ -51,11 +51,14 @@ object SchemaConverters {
    *
    * @since 2.4.0
```

Comment:
let's not mess up the API doc here.

Reply:
You mean I need to change it to since 2.5.0?

Reply:
no, this API doc is for `def toSqlType(avroSchema: Schema)` but not the new `def toSqlType(avroSchema: Schema, useStableIdForUnionType: Boolean)`

Reply:
I see!